src/components/navbar.component.tsx

```diff
@@ -78,6 +78,22 @@ const Navbar: React.FC<NavbarProps> = ({ user, onLogout, allowedLocales }) => {
               <Logo />
             </HeaderLink>
             <HeaderGlobalBar>
+              <ExtensionSlot extensionSlotName="add-patient-slot">
```

Comment:
Slot should be named "top-nav-actions-slot".

**Never** name the slot after what you *think* will be in there. Name the slot after the **location** where its placed.